spec/pmImageEditorSpec.js

```diff
@@ -384,6 +384,31 @@ describe('pmImageEditor', function () {
             expect(factory.initImageData).toHaveBeenCalledWith(200, 100);
         });
 
+        it('should allow to update selection if input is correct', function() {
+            spyOn(factory, 'resetTransformations').and.callThrough();
+
+            expect(factory.selection.width).toBe(150);
+            expect(factory.selection.height).toBe(75);
+
+            factory.updateSelectionDimensions(110, 120);
+
+            expect(factory.selection.width).toBe(110);
+            expect(factory.selection.height).toBe(120);
+
+            expect(factory.resetTransformations).toHaveBeenCalled();
+        });
+
+        it('should skip update selection if input is incorrect', function() {
```

Comment:
I think it would be better to explain *why* the input given is invalid (the width and height are greater than the image's own dimensions) -- it saves other devs some mental effort.